bin/build-constraints.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -eu
+
+runFromHook=${1:-"false"}
```

Comment:
I wish there was a bash formatter so we don't have such inconsistent code. Single quotes is always preferred when no need to expand anything.